shopsite/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en" ng-app="shopApp">
+<head>
+	<meta charset="UTF-8">
```

Comment:
looks like too many spaces as tab here. Could you please setup your ide to make tabs as spaces and 2 spaces per tab?